server/games/game.py

```diff
@@ -703,12 +753,17 @@ def clear_slot(self, slot_index):
 
         query_args = []
         for player in self.players:
-            player_option = functools.partial(self.get_player_option, player.id)
-            options = {key: player_option(key)
-                       for key in ['Team', 'StartSpot', 'Color', 'Faction']}
+            player_option = functools.partial(
+                self.get_player_option, player.id
+            )
+            options = {
+                key: player_option(key)
+                for key in ['Team', 'StartSpot', 'Color', 'Faction']
+            }
 
             def is_observer() -> bool:
-                return options.get('Team', -1) < 0 or options.get('StartSpot', 0) < 0
+                return options.get('Team',
```

Comment:
Yikes

Reply:
Yikes indeed. I see your point about linting the linter, although that of course somewhat defeats the purpose :/